Kickstarter-iOS/Library/WebViewController.swift

```diff
@@ -9,6 +9,8 @@ internal class WebViewController: UIViewController {
   internal let webView = WKWebView(frame: .zero, configuration: WKWebViewConfiguration())
   internal var bottomAnchorConstraint: NSLayoutConstraint?
 
+  var webKitCookieStore: WKHTTPCookieStore?
```

Comment:
This variable can be private

Reply:
Added a comment above the variable to explain what it's doing as well.